### DISCUSSION TITLE
How do i open the fooocus page?

### DISCUSSION THREAD

**Discussion by H1beer** [lllyasviel/Fooocus#2751] [Q&A] [answered]
It didnt open automatically

**Comment by Rogala** [comment]
[Any browser](http://127.0.0.1:7865/)

**Comment by ILikeToasters** [comment, marked as answer]
If it doesn't open automatically you can enter the default address into the address bar of your browser.  
<img width="229" alt="Screenshot 2024-04-11 195834" src="https://github.com/lllyasviel/Fooocus/assets/4357657/99e1096c-3f0e-47c2-a61c-8e1a43341d03">
